label_studio/core/utils/contextlog.py

```diff
@@ -271,7 +271,7 @@ def create_payload(self, request, response, body):
         else:
             namespace = request.resolver_match.namespace if request.resolver_match else None
             status_code = response.status_code
-            content_type = response.content_type
+            content_type = response.content_type if hasattr(response, 'content_type') else None
```

Comment:
nitpick (you can dismiss this):

```suggestion
            content_type = getattr(response, 'content_type', None)
```